src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,19 +173,60 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   ])
   const isFilled = useMemo(() => isOrderFilled(order), [order])
   // Display isLowBalance warning only for active and partial fill orders
-  const isLowBalance = isOverBalance && !isUnlimited && !isFilled && !isScheduled && !isPendingOrder && !isExpiredOrder
+  const isLowBalance =
+    isOverBalance &&
+    !isUnlimited &&
+    !isFilled &&
+    !isScheduled &&
+    !isActiveNextBatch &&
+    !isPendingOrder &&
+    !isExpiredOrder
 
   const pending = useMemo(() => isPendingOrder && <PendingLink transactionHash={transactionHash} />, [
     isPendingOrder,
     transactionHash,
   ])
 
+  // Dima's trick to force component update
+  const [, forceUpdate] = useSafeState({})
+
+  const refreshTimeout = useRef<null | NodeJS.Timeout>(null)
+
+  // Sets a timeout to update at least once at the end of each countdown
+  useEffect(() => {
+    function clear(): void {
+      if (refreshTimeout.current) clearTimeout(refreshTimeout.current)
+      refreshTimeout.current = null
+    }
+
+    clear()
+
+    const ms = getTimeRemainingInBatch(true)
+    console.log(`timeout useEffect activated, ${ms}ms left`)
+
+    if (isActiveNextBatch) {
+      refreshTimeout.current = setTimeout(() => forceUpdate({}), ms)
+    } else if (isFirstActiveBatch) {
+      refreshTimeout.current = setTimeout(() => forceUpdate({}), Math.max(0, ms - 60 * 1000))
```

Comment:
Presumably you don't need to schedule an update if `Math.max(0, ms - 60 * 1000) === 0`

Reply:
So in that case I just execute it straight away? Or skip it entirely?

Reply:
Straight away I think, hard to decide if that extra update will ultimately be useless, better to update